packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1470,6 +1471,14 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.widgets.SelectionOverlay.onSelectionHandleTapped}
   final VoidCallback? onSelectionHandleTapped;
 
+  /// {@template flutter.widgets.editableText.groupId}
+  /// The group identifier for the [TextFieldTapRegion] of this text field.
+  ///
+  /// Text fields with the same group identifier share the same tap region.
+  /// Defaults to the type of [EditableText].
```

Comment:
You'll want to add a "See also:" here that points to `TextFieldTapRegion`, and probably add the same `groupId` optional argument there, although you could also explain here that they could also use a raw `TapRegion` with the `groupId` at that point, since they are entirely equivalent. Adding `groupId` to `TextFieldTapRegion` feels a little weird because it basically turns it into `TapRegion`, but without as many options. This is probably OK, given it is meant for a specific use case, but technically odd.

Reply:
Your suggestion has been received